basin3d/plugins/usgs.py

```diff
@@ -125,14 +126,23 @@ def generator_usgs_measurement_timeseries_tvp_observation(view,
 
```

Comment:
I believe aggregation_duration should always be in the query args b/c of how we are handling in core/synthesis.py. However, now that we have are broadening support, I suggest verifying that aggregation_duration is either NONE or DAY since that is all we are supporting currently in this plugin. 

In future If the param become truly optional, we'll need to handle the case in which aggregation_duration is not in the query args.